project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/filter/filter.inc

```diff
@@ -47,10 +47,11 @@ function c4m_features_homepage_filter_form_create() {
     $options['projects'] = t('My projects');
   }
 
-  $options += array(
-    'interests' => t('My interests'),
-    'showAll'   => t('Show all'),
-  );
+  if (c4m_user_profile_get_interests()) {
+    $options['interests'] = t('My interests');
+  }
+
+  $options['showAll'] = t('Show all');
```

Comment:
@ybaras 
good catch, please return early if `$options` is empty, see:
https://github.com/capacity4dev/capacity4more/pull/1466/commits/acbbd2cd33985847d0dc5e42813f70eb0e45f684